.github/workflows/ci-pipeline.yml

```diff
@@ -27,12 +27,16 @@ jobs:
       -
         uses: actions/checkout@v2
       - name: Run unit and integration tests
-        run: docker-compose -f docker-compose.test-ci.yml run test-ci
+        run: | 
+          docker-compose -f docker-compose.test-ci.yml run \
+          -e LOG_LEVEL=DEBUG \
+          test-ci
         if: github.event.schedule != '08 13 * * *'
       - 
         name: E2E test app terraform init
         working-directory: terraform/modules/test
         run: terraform init
+        if: github.event.schedule != '08 13 * * *' && github.event_name == 'pull_request' && github.base_ref == 'main'
```

Comment:
Not one I'm certain on but given that you check the event name is `pull_request`, do we _also_ need to check that it's not the schedule?

Regardless, sensible approach to prevent the TF locking issue 👍